themes/jaeger-docs/layouts/partials/javascript.html

```diff
@@ -4,13 +4,15 @@
 {{ $jsFiles := (slice "jquery-ui-1.9.1.custom.min.js" "anchor.min.js" "tocbot.min.js" "app.js") }}
 {{ range $jsFiles }}
 {{ $js := resources.Get (printf "js/%s" .) | minify | fingerprint }}
-<script src="{{ $js.Permalink }}" integrity="{{ $js.Data.Integrity }}"></script>
+<script src="{{ $js.RelPermalink }}" integrity="{{ $js.Data.Integrity }}"></script>
 {{ end }}
 
+{{/*
```

Comment:
please elaborate

Reply:
AFAIK, the website doesn't use Lurn, so it shouldn't be including Lunr JS.

Reply:
Oh, I missed that when I was searching the code for a search layout or partial! Let me reinstate the Lunr JS then.